diff -ur smokeping-2.8.2.org/bin/smokeinfo smokeping-2.8.2/bin/smokeinfo
--- smokeping-2.8.2.org/bin/smokeinfo	2021-08-10 12:00:33.000000000 +0200
+++ smokeping-2.8.2/bin/smokeinfo	2023-04-02 18:16:09.223051686 +0200
@@ -2,8 +2,7 @@
 use strict;
 use warnings;
 
-use lib (split /:/, q{}); # PERL5LIB
-use FindBin;use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
+use lib "/usr/share/smokeping";
 
 use Smokeping::Info;
 use Getopt::Long 2.25 qw(:config no_ignore_case);
diff -ur smokeping-2.8.2.org/bin/smokeping smokeping-2.8.2/bin/smokeping
--- smokeping-2.8.2.org/bin/smokeping	2021-08-13 08:11:39.000000000 +0200
+++ smokeping-2.8.2/bin/smokeping	2023-04-02 18:16:09.223051686 +0200
@@ -4,13 +4,11 @@
 use strict;
 use warnings;
 
-use FindBin;
-use lib (split /:/, q{}); # PERL5LIB
-use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
+use lib "/usr/share/smokeping";
 
 use Smokeping;
  
-Smokeping::main("$FindBin::RealBin/../etc/config");
+Smokeping::main("/etc/smokeping/config");
 
 =head1 NAME
 
diff -ur smokeping-2.8.2.org/bin/smokeping_cgi smokeping-2.8.2/bin/smokeping_cgi
--- smokeping-2.8.2.org/bin/smokeping_cgi	2021-08-13 08:11:39.000000000 +0200
+++ smokeping-2.8.2/bin/smokeping_cgi	2023-04-02 18:16:58.269718358 +0200
@@ -4,9 +4,7 @@
 use strict;
 use warnings;
 
-use FindBin;
-use lib (split /:/, q{}); # PERL5LIB
-use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
+use lib "/usr/share/smokeping";
 
 # don't bother with zombies
 $SIG{CHLD} = 'IGNORE';
@@ -17,7 +15,7 @@
 
 use CGI::Fast;
 
-my $cfg = (shift @ARGV) || "$FindBin::Bin/../etc/config";
+my $cfg = (shift @ARGV) || "/etc/smokeping/config";
 
 
 while (my $q = new CGI::Fast) {
diff -ur smokeping-2.8.2.org/bin/tSmoke smokeping-2.8.2/bin/tSmoke
--- smokeping-2.8.2.org/bin/tSmoke	2021-08-13 08:11:39.000000000 +0200
+++ smokeping-2.8.2/bin/tSmoke	2023-04-02 18:16:09.223051686 +0200
@@ -52,9 +52,7 @@
 #
 # Point the lib variables to your implementation
 
-use lib (split /:/, q{}); # PERL5LIB
-use FindBin;use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
-
+use lib "/usr/share/smokeping";
 
 use Smokeping;
 use Net::SMTP;
@@ -63,7 +61,7 @@
 use RRDs;
 
 # Point to your Smokeping config file
-my $cfgfile = (shift @ARGV) || "$FindBin::Bin/../etc/config";
+my $cfgfile = (shift @ARGV) || "/etc/smokeping/config";
 
 # global variables
 my $cfg;
diff -ur smokeping-2.8.2.org/etc/basepage.html.dist smokeping-2.8.2/etc/basepage.html.dist
--- smokeping-2.8.2.org/etc/basepage.html.dist	2019-09-23 09:46:17.000000000 +0200
+++ smokeping-2.8.2/etc/basepage.html.dist	2023-04-02 18:16:09.226385019 +0200
@@ -4,8 +4,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <META http-equiv="Refresh" content="<##step##>">
     <title>SmokePing Latency Page for <##title##></title>
-    <link rel="stylesheet" type="text/css" href="css/smokeping-print.css" media="print">
-    <link rel="stylesheet" type="text/css" href="css/smokeping-screen.css" media="screen">
+    <link rel="stylesheet" type="text/css" href="/smokeping/css/smokeping-print.css" media="print">
+    <link rel="stylesheet" type="text/css" href="/smokeping/css/smokeping-screen.css" media="screen">
 </head>
 <body id="body">
 
@@ -47,10 +47,10 @@
     <p><small>Maintained by <a href="mailto:<##contact##>"><##owner##></a></small></p>
 </div>
 
-<script src="js/prototype.js" type="text/javascript"></script>
-<script src="js/scriptaculous/scriptaculous.js?load=builder,effects,dragdrop" type="text/javascript"></script>
-<script src="js/cropper/cropper.js" type="text/javascript"></script>
-<script src="js/smokeping.js" type="text/javascript"></script>
+<script src="/smokeping/js/prototype.js" type="text/javascript"></script>
+<script src="/smokeping/js/scriptaculous/scriptaculous.js?load=builder,effects,dragdrop" type="text/javascript"></script>
+<script src="/smokeping/js/cropper/cropper.js" type="text/javascript"></script>
+<script src="/smokeping/js/smokeping.js" type="text/javascript"></script>
 
 </body>
 </html>
diff -ur smokeping-2.8.2.org/lib/Smokeping/Examples.pm smokeping-2.8.2/lib/Smokeping/Examples.pm
--- smokeping-2.8.2.org/lib/Smokeping/Examples.pm	2011-10-05 11:51:46.000000000 +0200
+++ smokeping-2.8.2/lib/Smokeping/Examples.pm	2023-04-02 18:16:42.016385025 +0200
@@ -12,7 +12,7 @@
 This module generates L<smokeping_examples> and the example
 configuration files distributed with Smokeping. It is supposed to be
 invoked from the smokeping distribution top directory, as it will need
-the C<etc/config.dist> template configuration file and will create files
+the C</etc/smokeping/config> template configuration file and will create files
 in the directories C<doc> and C<doc/examples>.
 
 =head1 DESCRIPTION
@@ -107,7 +107,7 @@
 ${e}head1 DESCRIPTION
 
 Currently the examples differ only in the C<Probes> and C<Targets>
-sections. The other sections are taken from the C<etc/config.dist>
+sections. The other sections are taken from the C</etc/smokeping/config>
 configuration template in the Smokeping distribution so that the example
 files are complete.
 
diff -ur smokeping-2.8.2.org/lib/Smokeping.pm smokeping-2.8.2/lib/Smokeping.pm
--- smokeping-2.8.2.org/lib/Smokeping.pm	2021-08-13 08:19:27.000000000 +0200
+++ smokeping-2.8.2/lib/Smokeping.pm	2023-04-02 18:16:09.226385019 +0200
@@ -3871,7 +3871,7 @@
 
  *** Probes ***
  +FPing
- binary = /usr/sepp/bin/fping
+ binary = /usr/bin/fping
 
 You can override it for a particular slave like this:
 
