diff -urN smokeping-2.7.3.org/bin/smokeinfo smokeping-2.7.3/bin/smokeinfo
--- smokeping-2.7.3.org/bin/smokeinfo	2018-04-13 14:13:05.000000000 +0200
+++ smokeping-2.7.3/bin/smokeinfo	2019-03-15 09:15:16.281298555 +0100
@@ -2,8 +2,7 @@
 use strict;
 use warnings;
 
-use lib (split /:/, q{}); # PERL5LIB
-use FindBin;use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
+use lib "/usr/share/smokeping";
 
 use Smokeping::Info;
 use Getopt::Long 2.25 qw(:config no_ignore_case);
diff -urN smokeping-2.7.3.org/bin/smokeping smokeping-2.7.3/bin/smokeping
--- smokeping-2.7.3.org/bin/smokeping	2018-04-13 14:13:05.000000000 +0200
+++ smokeping-2.7.3/bin/smokeping	2019-03-15 09:13:44.808555196 +0100
@@ -4,13 +4,11 @@
 use strict;
 use warnings;
 
-use lib (split /:/, q{}); # PERL5LIB
-use FindBin;
-use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
+use lib "/usr/share/smokeping";
 
 use Smokeping;
  
-Smokeping::main("$FindBin::RealBin/../etc/config");
+Smokeping::main("/etc/smokeping/config");
 
 =head1 NAME
 
diff -urN smokeping-2.7.3.org/bin/smokeping_cgi smokeping-2.7.3/bin/smokeping_cgi
--- smokeping-2.7.3.org/bin/smokeping_cgi	2018-04-13 14:13:05.000000000 +0200
+++ smokeping-2.7.3/bin/smokeping_cgi	2019-03-15 09:14:55.490675372 +0100
@@ -4,9 +4,7 @@
 use strict;
 use warnings;
 
-use lib (split /:/, q{}); # PERL5LIB
-use FindBin;
-use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
+use lib "/usr/share/smokeping";
 
 # don't bother with zombies
 $SIG{CHLD} = 'IGNORE';
@@ -17,8 +15,7 @@
 
 use CGI::Fast;
 
-my $cfg = (shift @ARGV) || "$FindBin::Bin/../etc/config";
-
+my $cfg = (shift @ARGV) || "/etc/smokeping/config";
 
 while (my $q = new CGI::Fast) {
    Smokeping::cgi($cfg,$q);
diff -urN smokeping-2.7.3.org/bin/tSmoke smokeping-2.7.3/bin/tSmoke
--- smokeping-2.7.3.org/bin/tSmoke	2018-04-13 14:13:05.000000000 +0200
+++ smokeping-2.7.3/bin/tSmoke	2019-03-15 09:14:23.476382053 +0100
@@ -52,9 +52,7 @@
 #
 # Point the lib variables to your implementation
 
-use lib (split /:/, q{}); # PERL5LIB
-use FindBin;use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
-
+use lib "/usr/share/smokeping";
 
 use Smokeping;
 use Net::SMTP;
@@ -63,7 +61,7 @@
 use RRDs;
 
 # Point to your Smokeping config file
-my $cfgfile = "etc/config.dist";
+my $cfgfile = "/etc/smokeping/config";
 
 # global variables
 my $cfg;
diff -urN smokeping-2.7.3.org/doc/smokeping_install.pod smokeping-2.7.3/doc/smokeping_install.pod
--- smokeping-2.7.3.org/doc/smokeping_install.pod	2018-01-30 21:40:07.000000000 +0100
+++ smokeping-2.7.3/doc/smokeping_install.pod	2019-03-15 09:11:30.687864995 +0100
@@ -108,7 +108,7 @@
 
 =head2 Configuration
 
-Use the F<etc/config.dist> file as a template to create your own smokeping configuration file.
+Use the F</etc/smokeping/config> file as a template to create your own smokeping configuration file.
 See L<smokeping_config> for details.
 
 =over
diff -urN smokeping-2.7.3.org/lib/Smokeping/Examples.pm smokeping-2.7.3/lib/Smokeping/Examples.pm
--- smokeping-2.7.3.org/lib/Smokeping/Examples.pm	2011-10-05 11:51:46.000000000 +0200
+++ smokeping-2.7.3/lib/Smokeping/Examples.pm	2019-03-15 09:11:30.687864995 +0100
@@ -12,7 +12,7 @@
 This module generates L<smokeping_examples> and the example
 configuration files distributed with Smokeping. It is supposed to be
 invoked from the smokeping distribution top directory, as it will need
-the C<etc/config.dist> template configuration file and will create files
+the C</etc/smokeping/config> template configuration file and will create files
 in the directories C<doc> and C<doc/examples>.
 
 =head1 DESCRIPTION
@@ -107,7 +107,7 @@
 ${e}head1 DESCRIPTION
 
 Currently the examples differ only in the C<Probes> and C<Targets>
-sections. The other sections are taken from the C<etc/config.dist>
+sections. The other sections are taken from the C</etc/smokeping/config>
 configuration template in the Smokeping distribution so that the example
 files are complete.
 
diff -urN smokeping-2.7.3.org/lib/Smokeping.pm smokeping-2.7.3/lib/Smokeping.pm
--- smokeping-2.7.3.org/lib/Smokeping.pm	2018-12-20 11:24:44.000000000 +0100
+++ smokeping-2.7.3/lib/Smokeping.pm	2019-03-15 09:13:01.330584016 +0100
@@ -3620,7 +3620,7 @@
 
  *** Probes ***
  +FPing
- binary = /usr/sepp/bin/fping
+ binary = /usr/bin/fping
 
 You can override it for a particular slave like this:
 
