diff -ur smokeping-2.8.2.org/Makefile.am smokeping-2.8.2/Makefile.am
--- smokeping-2.8.2.org/Makefile.am	2021-08-13 08:11:43.000000000 +0200
+++ smokeping-2.8.2/Makefile.am	2023-04-02 18:10:54.489718313 +0200
@@ -16,7 +16,7 @@
 
 AUTOMAKE_OPTIONS =  foreign
 
-SUBDIRS = lib thirdparty bin doc etc htdocs
+SUBDIRS = lib bin doc etc htdocs
 
 EXTRA_DIST = COPYRIGHT CHANGES CONTRIBUTORS LICENSE cpanfile VERSION README.md
 
